ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -212,3 +216,19 @@ const statusIconTextMap: Record<ComplianceCheckStatus, ClusterStatusObject> = {
 export function getClusterResultsStatusObject(status: ComplianceCheckStatus): ClusterStatusObject {
     return statusIconTextMap[status];
 }
+
+export function createScanConfigFilter(selectedScanConfig: QueryValue) {
+    return typeof selectedScanConfig === 'string'
```

Comment:
As I started reviewing from end of changed files, this line answered a question in the back of my mind:
* is `selectedScanConfig` the name (most likely hypothesis from use)
* or the object

What are pro and con to add `Name` to the end of the name (pardon pun).

Reply:
`Name` should definitely be included for clarity. Will update. 

Although this also makes me wonder if there's an alternative approach to using the `useURLParameter` hook. It would be nice to guarantee `string | undefined` at least for the context value being passed around as opposed to `QueryValue` but I can see an argument being made in favor of `QueryValue`. The `useURLStringUnion` is appealing too but I don't think it's intended to be used with network data.